### PR TITLE
Refactor parser

### PR DESCRIPTION
**Pull request by serucee**

### CODE REVIEW

File: app/Helper/FileParserJson.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace ArtifactCreation\Helper;
+
+use ArtifactCreation\Exception\InvalidFileException;
+use ArtifactCreation\Exception\MissingFileException;
+
+/**
+ * Parse json file to array
+ *
+ * Class Parser
```
Comment (ozada) [inline app/Helper/FileParserJson.php]: Class FileParserJson

File: app/Helper/FileParserJson.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     * Parse json file to array
+     *
+     * @return array
+     *
+     * @throws InvalidFileException
+     * @throws MissingFileException
+     */
+    public function parse()
+    {
+        $fullyQualifiedFileName = $this->file;
```
Comment (ozada) [inline app/Helper/FileParserJson.php]: You ca use $this->file direct instead of assign it to $fullyQualifiedFileName

File: app/main.php [hunk 3]
```diff
@@ -3,16 +3,17 @@
...
 
-use ArtifactCreation\Helper\ConfigurationParser;
+use ArtifactCreation\Model\Configuration;
 use ArtifactCreation\Helper\ExceptionHelper;
+use ArtifactCreation\Helper\FileParserJson;
 use ArtifactCreation\Helper\Runner;
 use Exception;
 
 require __DIR__ . '/vendor/autoload.php';
 $fullyQualifiedConfigurationFileName = '/github/workspace/.github/artifact-configuration.json';
```
Comment (ozada) [inline app/main.php]: Leave here comment regarding why its so hardcoded.

File: app/Helper/FileParserJson.php (hunk 2 above)
Comment (serucee) [reply app/Helper/FileParserJson.php]: Copy pasta ;D

**Pull request merged**